### PR TITLE
Add search grounding quickstarts for python+rest.

### PR DESCRIPTION
**Pull request by MarkDaoust**

### CODE REVIEW

File: quickstarts/Search_Grounding.ipynb [hunk 1]
```diff
@@ -0,0 +1,421 @@
+{
```
Comment (markmcd) [inline quickstarts/Search_Grounding.ipynb:1]: <p>If the package is out when you merge, can you update this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (markmcd) [inline quickstarts/Search_Grounding.ipynb:1]: <p>Duplicate line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (markmcd) [inline quickstarts/Search_Grounding.ipynb:1]: <p>Is it worth pointing out that there are requirements around this field? <a href="https://googledevai.devsite.corp.google.com/gemini-api/docs/grounding/search-suggestions?hl=en#requirements" target="_blank">e.g.</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

File: quickstarts/rest/Search_Grounding.ipynb [hunk 2]
```diff
@@ -0,0 +1,267 @@
+{
```
Comment (markmcd) [inline quickstarts/rest/Search_Grounding.ipynb:1]: <p>IMAGEN_API_KEY</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (markmcd) [inline quickstarts/rest/Search_Grounding.ipynb:1]: <p>Same here, re: pointing out that there are requirements.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

**Review by markmcd — DISMISSED** [review]
Just a couple of minor things. As always they don't have to be blockers.

File: quickstarts/Search_Grounding.ipynb (hunk 1 above)
Comment (MarkDaoust) [reply quickstarts/Search_Grounding.ipynb:1]: <p>Done.</p>

Comment (MarkDaoust) [reply quickstarts/Search_Grounding.ipynb:1]: <p>Done.</p>

File: quickstarts/rest/Search_Grounding.ipynb (hunk 2 above)
Comment (MarkDaoust) [reply quickstarts/rest/Search_Grounding.ipynb:1]: <p>Done.</p>

Comment (MarkDaoust) [reply quickstarts/rest/Search_Grounding.ipynb:1]: <p>Done.</p>

**Pull request merged**